pkg/cmd/gist/view/view_test.go

```diff
@@ -328,3 +331,90 @@ func Test_viewRun(t *testing.T) {
 		})
 	}
 }
+
+func Test_promptGists(t *testing.T) {
```

Comment:
these are good tests :+1: please also add one test to the `gist view` tests just to ensure that we go into interactive mode when no ID is supplied (it doesn't have to test anything beyond that since you're doing that here).

Reply:
Tests added. Could you please have a look? Thanks.